src/screens/PluginStore/PluginModal.tsx

```
@@ -326,55 +337,61 @@ const PluginModal: React.FC<IPluginModalProps> = ({
               {/* Tabs */}
               <div className={styles.tabsContainer}>
-                {TABS.map((t) => (
+                {TABS.map((tabKey) => (
                   <div
-                    key={t}
-                    onClick={() => setTab(t)}
-                    className={tab === t ? styles.tabActive : styles.tab}
+                    key={tabKey}
+                    onClick={() => setTab(tabKey)}
+                    className={tab === tabKey ? styles.tabActive : styles.tab}
                   >
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)